### DISCUSSION TITLE
Unicode characters

### DISCUSSION THREAD

**Discussion by iamvdo** [cssbattle/forum#9] [Ideas]
Hello everyone.

Many players don't think unicode characters are a great option (including me at some point). But, it's a pretty tough subject. **So, how would you handle it?**

There was a subject on previous platform where some edge cases were mentioned: https://spectrum.chat/css-battle/general/unicode-solutions-restriction-fix-or-remove~0dda805f-ebf8-4037-a219-0dcf68a9deaf

This was my last message on the mentioned post, and I'm still OK with that:

>I thought again about this, and my opinion changed a bit. I think unicode solutions are still CSS solutions, and banned them in some ways would only lead us finding alternatives, or set too much restrictive rules. I remember @anwerso suggesting adding a badge (next to score) saying that a solution is using unicode/text. It could be detected automatically, or be chosen by the user to inform other players. In that regard, the UI could allow unicode as-is characters, and you (CSSBattle creators) will even be able to add specific preview (as @alexzaworski's preview tool) right from the UI (for PRO users for example). In the same approach, other kind of badges can be created (better gamification) or even badges for different browsers (pupetteer can launch firefox!). At the end, leaderboards can be filtered using these badges.

We should talk about it, and find a "not so bad" solution :)

**Comment by Beowolve** [comment]
The only reason why I am against it, is because the rendering is different on the server. Using Unicode would be fine if we could get a better way of reproducing the solutions.
How about setting up a virtual machine for VirtualBox with a browser setup like on the server, so that the solution looks exactly the same. That way everyone has equal chances.

I tried Ubuntu 18.04 with Chromium, still doesn't work for me the rendering is different. What is your setup to create Unicode solutions? You can't be guessing the right numbers...

SVG are not allowed, but have been in the past as I understand. What is the reasoning behind that, I have started doing CSSBattle just 2 weeks ago, I don't know what happened back then.
